core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -210,6 +210,10 @@ export class DotEmaShellComponent implements OnInit, OnDestroy {
             params.clientHost = uveConfig.url;
         }
 
+        if (params.preview && !params.publishDate) {
+            params.publishDate = new Date().toISOString();
+        }
```

Comment:
Are we setting `publishDate` all the time as soon as user goes to preview

Reply:
Yes, I set it so that if you enter preview mode, the page is loaded with the current date by default
Isn't that the behavior we want?

So if the link have preview but dont have a publishDate, we set the current date too